data/json/recipes/other/containers.json

```
@@ -152,8 +152,7 @@
     "difficulty": 3,
     "time": "2 h",
     "autolearn": true,
-    "using": [ [ "blacksmithing_intermediate", 2 ] ],
-    "tools": [ [ [ "crucible", -1 ], [ "crucible_clay", -1 ] ], [ [ "swage", -1 ] ] ],
+    "using": [ [ "blacksmithing_advanced", 2 ] ],
```

Was this meant to have been upgraded from `blacksmithing_intermediate` to advanced for hip flasks? If not:
```suggestion
    "using": [ [ "blacksmithing_intermediate", 2 ] ],
```

---

Yee, makes sense now.